### DISCUSSION TITLE
Everything Is Chrome

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#11] [Comments Archive]
# Everything Is Chrome

My look into the overarching dominance of Google Chrome and it's overwhelmingly negative impact on the web as we know it.

https://vale.rocks/posts/everything-is-chrome

**Comment by brycewray** [comment]
Great work. As I told you in replying to your comment about my article:

> Thanks for the kind words, the reference, **and** — not the least — your own article! And, yes, the potential near-future experience of web browsing is troubling given that, as the saying goes, “Those who cannot remember the past are doomed to repeat it,” and there would seem to be too many folks in high places who have all-too-short memories. 😞

**Comment by lagraviereScience** [comment]
Just a little mistake I noticed: "and people were more then interested " should be "and people were more than interested"
